api/v1alpha7/types.go

```diff
@@ -163,6 +163,24 @@ type RootVolume struct {
 	AvailabilityZone string `json:"availabilityZone,omitempty"`
 }
 
+type AdditionalBlockDevice struct {
+	// NameSuffix is the suffix to be added to volume names.
+	// It will be combined with the machine name to make the volume name.
+	NameSuffix string `json:"nameSuffix"`
```

Comment:
Could we call this `name` instead? The docstring can explain that the volume name will be the machine name with this appended.

Reply:
I don't understand why you want to name it `Name`. This is not a name, this is a name suffix.
This would be a lie to the API user that their block device would be named like this, because this wouldn't be the case.

Reply:
See below. It would be the name of the volume *in the context of this machine*. It would also be the name of the volume in the `additionalBlockDevices` list.

Reply:
fair enough, I'll do it.